ytsaurus-ui-chart/values.yaml

```diff
@@ -0,0 +1,81 @@
+ui:
+  replicaCount: 1
+  image:
+    repository: ytsaurus/ui
+    pullPolicy: IfNotPresent
+    tag: ""
+
+  clusterConfig:
+    clusters: []
+      # - authentication: basic
+      #   description: My first YTsaurus. Handle with care.
+      #   environment: testing
+      #   group: My YTsaurus clusters
+      #   id: minisaurus
+      #   name: minisaurus
+      #   primaryMaster:
+      #     cellTag: 1
+      #   proxy: http-proxies-lb.testing.svc.cluster.local
+      #   secure: false
+      #   theme: lavander
+
+  env:
+    - name: ALLOW_PASSWORD_AUTH
+      value: "1"
+
+  secrets:
+    interfaceSecrets:
+      # see https://github.com/ytsaurus/ytsaurus-ui/tree/main/packages/ui#how-to-work-with-the-repo
+      secretName: "yt-ui-secret"
+      secretKey: "yt-interface-secret.json"
+
+service:
+  type: NodePort
+  port: 80
+
+livenessProbe:
+  httpGet:
+    path: /
+    port: http
+readinessProbe:
+  httpGet:
+    path: /
+    port: http
+
+ngnix:
+  useCustomConfig: false
+  # name of config map where custom config for nginx is located 
+  configMapName: "custom-nginx-configmap"
+  configMapkey: "ngnix.conf"
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: Prefix
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+imagePullSecrets: []
+
+nameOverride: ""
+fullnameOverride: ""
+
+
+settings:
```

Comment:
Can you add ytOAuthSettings for support [this](https://github.com/ytsaurus/ytsaurus-ui/blob/main/packages/ui/docs/configuration.md#oauth)?

And here https://github.com/ytsaurus/ytsaurus-ui/pull/544/files#diff-c619831eebf125c84bb6f25eefd3b73c419032c67767f8b59dacda4a37c06999R11-R12
Something like this - 
```yaml
data:
  common.js: |
    module.exports = {
      odinBaseUrl: {{ .Values.settings.odinBaseUrl | quote }},
      uiSettings: {
        directDownload: {{ .Values.settings.directDownload | toJson }}
      },
      ytOAuthSettings: {{ .Values.settings.ytOAuthSettings }}
    }
```

It seems to be working. - [helm-playground](https://helm-playground.com/#t=LQhQEMAcEsDUFMBOBnaB7AdgLgAQDcBGUAa2gwBNcBhTAM2gHMBZKUAOg9HPABdwtQOHAGM0AWzGY2AK2S4APoKE5J5AK4AbeG3gAPSGkQ9kOALw4A3kuU405MgCFwyeAFVEG3BYs42scBpq8MhsLjw8ZAwhdo7Obh448jgAjmpoPPA4AL5ZADTWymrQAMrw4ZFylgU29ojwwjwAImgA7hgaaOCUlj5%2BAUEhYREYUWy19U2t7Z3kiTg8aABSyJjZWdXZ%2BTZCAJ48APIAgmo8ABal5SOV3r7%2BgcGhZcOje0cn508VawXrQA&v=LQhQGcFMBdoSwHYHNwC5QAIMHsAmiAhAQygFUAnAG1QwCIAPR2zDfcyAY2gBFsB3BJWxFcNaOQCukFgE9oAeQCCE6AAsAyjHjI0GAD7AWWAN5Gs5i7Q6U4kBNACSuWjQAO5bB0jhwAOjsAbr7yAAoAogByDtwA%2BgDCADIOkQAqMdEANGYWlta29poc7NAuGO6e3n6BweFRsephcQBKYSnZOQC%2BoEA)